src/App.tsx

```diff
@@ -2,105 +2,37 @@ import React from 'react';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { PostList } from './components/PostList';
+import { User } from './components/User';
+import { Comment } from './components/Comment';
+import { Post } from './components/Post';
+
+function getUser(userId: number): User | null {
```

Comment:
If you are searching some entity by some filter, specify it in function/method name [[checklist]](https://github.com/mate-academy/react_static-list-of-posts/blob/master/checklist.md)
```suggestion
function getUserById(userId: number): User | null {
```